.github/workflows/code-check.yml

```diff
@@ -22,6 +22,7 @@ on:
         description: Token Report to Codecov
 
 env:
+  NX_PARALLEL: ${{ vars.NX_PARALLEL }}
```

Comment:
any reason to use a repository variable instead of an environment variable directly ?